modules/rosa-architecture.adoc

```diff
@@ -0,0 +1,13 @@
+[id="rosa-architecture_{context}"]
+= ROSA architecture on public and private networks
+
+You can install ROSA using either a public or private network.
```

Comment:
Delete line breaks to create a single paragraph with 4, 6, and 8. Delete _You can_ and begin the first sentence with _Install_.

Reply:
I'm hesitant to make that change because there is a third installation option (PrivateLink). So I think we should leave this one as "You can install".